### ISSUE TITLE
Config suggestion

### ISSUE THREAD

**Issue by YuroKashine** [BudgetAirpods/InsanityRemastered#27]
Hi i love this mod
However i would love to have more say in how fast sanity regularly increases over time. even when i downgrade the sanity loss configs it still goes down pretty fast. And the sanity gain doesnt seem to work... it might be because default values are minus when the config says gain... it did look like i lost more sanity while being near my buddy.
Furthermore itd be nice to be able to configure how much sanity gain a flashlight and and or lightsource gives when its active.

This way i could totally tweak how fast my sanity regularly goes down... how fast it will go down when in full dark... if my flashlight prevents some sort of sanity loss and i can have a say in how much... and tweak all the other stuffs.

I hope it could be implemented coz i really love the idea behind it

**Comment by YuroKashine** [comment]
Yes when i put sanity gain when near player into the plus i started gaining sanity. However the ship does have to stay negative tho because i instantly was full insane in the ship when i changed it🤣 im still testing the config for sanity loss now

**Comment by YuroKashine** [comment]
Also if full insane is there a sound in this mod that goes really loud like a loud BEEP sound? Trying to figure out which mod just scared the living daylights out of me

**Comment by BudgetAirpods** [comment]
The sanity gain/loss is something I'm still tweaking around as a lot of people are complaining about it. Sorry. 

As for the sound, if it sounds like some kind of alarm, then most likely yes.

**Comment by YuroKashine** [comment]
Lmfao are there configurables for that? I mean i love it but it scared the absolute f out of me🤣

**Comment by BudgetAirpods** [comment]
> Lmfao are there configurables for that? I mean i love it but it scared the absolute f out of me🤣

Kinda?? It disables all of the other sounds that will randomly play though
